.github/workflows/pythonapp.yml

```diff
@@ -22,40 +22,75 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         python-version: ['3.9', '3.10', '3.11', '3.12']
+        include:
+          - os: windows-latest
+            python-version: '3.11'
```

Comment:
Any reason to test with 3.11 rather than 3.12?

Reply:
Not really, just providing some coverage because I will run against 3.12 on Windows elsewhere, e.g. DOLFINx.